hw2/04-background.js

```diff
@@ -1 +1,42 @@
-// Add your code here
+const body = document.getElementById("viewport");
+const timerValue = document.getElementById("timerInput");
+const timerButton = document.getElementById("button");
+let interval = null;
+let timerOn = false;
+
+function generateColorPalette() {
+  const r = Math.floor(Math.random() * 200);
+  const g = Math.floor(Math.random() * 200);
+  const b = Math.floor(Math.random() * 200);
+  const colorCode = `rgb(${r}, ${g}, ${b})`;
```

Comment:
consider using hsla() colors instead of rgb and lowering the alpha value of it as suggested by professor

Reply:
Thanks for that point, it slipped my mind that we were suggested using hsla.
I have accordingly updated the color palette in my latest code changes!